app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
-  before_action :find_user, only: %i[index show like unlike]
-  before_action :find_post, only: %i[show like unlike]
+  before_action :find_user, only: %i[create index show like unlike destroy]
```

Comment:
- It would be a good idea to invoke the `cancan` gem. Please something like below;
```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
Done